ui/remoteAccess.js

```diff
@@ -0,0 +1,53 @@
+// -*- mode: js; js-indent-level: 4; indent-tabs-mode: nil -*-
+/* exported RemoteAccessApplet */
+/*
+ * Copyright © 2020 Endless OS Foundation LLC
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+const { GObject } = imports.gi;
+
+/* The upstream impl hides the indicator on non wayland sessions
+ * and given we only enable X11 on Endless OS atm let's specialize it
```

Comment:
This will stop being true very soon. For new aarch64 platforms, except Raspberry Pi, the plan is to use Wayland. Will the indicator show twice in that case?

Reply:
It won't show twice but I should probably disconnect the original signal (which upstream doesn't save the source id) to avoid running the callback twice (and also to avoid errors if the indicator gets destroyed - atm we don't destroy the indicators though).

I will update the PR to disconnect the original signal connection.

Reply:
Updated, pushed a new commit - I don't like it, it is a real hack but the alternative would be to duplicate the whole indicator code (which wouldn't be the end of the world given the original code is considerably small). Please let me know what you think and I can update again if needed.

Reply:
I don't have enough context to meaningfully review this I'm afraid.

Reply:
(It does indeed seem like a real hack!)